playbooks/technical-planning.md

```diff
@@ -0,0 +1,81 @@
+
+If product-related feedback comes up while the tech plan is being reviewed, that is often a sign that there is more to discuss and debate at the product brief-level. If you are authoring a tech plan and this happens, it is recommended that you bring the conversation immediately to the associated PM(s). Once the question or feedback is addressed, the tech planning can continue.
+
+## Tech Plan States
+### Draft
+A tech plan in this state is a work in progress.
+
+### Proposed
+A tech plan in this state is officially open for feedback. Anyone should feel free to look at Proposed tech plans and leave their thoughts.
+
+### Accepted
+A tech plan in this state has been approved by all associated stakeholders. At this stage, it can be implemented or deprioritized.
+
+### Closed
+A tech plan in this state has either been rejected outright (i.e. it was proposing a new technology that the team decides not to adopt) or otherwise abandoned before it was able to be formally accepted.
+
+### Implemented
+A tech plan in this state has been implemented. Generally, tech plans can be put into this state at the same time that the associated product brief, if there is one, is moved to "Done".
+
+## Writing a tech plan
+We recommend following the template in Notion, but feel free to add or remove sections. Be as clear, concise, and organized as possible when writing so that it is easy for others to understand and leave quality feedback.
+
+If the technical plan involves a contentious or impactful technical decision, we recommend creating a separate, linked, document to capture the alternatives considered ([example here](https://www.notion.so/artsy/d17290484e6b40ac9e871eb7070dd3e8?v=81f6f3bf425b43e48fadd3c36adc2e09)). This allows the tech plan to focus on the chosen option.
+
+As with the tech planning process itself, creating decision matrices benefits from [collaboration and feedback](https://review.firstround.com/this-matrix-helps-growing-teams-make-great-decisions).
+
+## Stakeholders
+Stakeholders represent the list of people from whom you require tech plan approval. Think of this as similar to the "Reviewers" field on Github PRs. As the author of the tech plan, you are responsible for choosing appropriate stakeholders to arrive at the best technical approach.
+
+Some guidelines for choosing stakeholders:
+- Aim for ~3-5 stakeholders.
+- Stakeholders should be from the engineering team, given the tech plan is focused on implementation decisions. Of course it's great to share and collaborate outside of engineering, but other team members need not be on this list.
+- Choose stakeholders whose expertise is important for ensuring the best decision is made. If the change affects another team's domain, include someone from that team. Consider including your team's tech lead as well as relevant staff engineers.
```

Comment:
Sometimes I volunteer feedback despite not being invited as a stakeholder. This could even be critical/blocking feedback. Should I add myself as a stakeholder in order to capture that status formally? (Or would that be off-putting to authors?)

Reply:
I think this _could_ continue to be a source of tension—no provision is made in the process for the possibility that someone outside of your stakeholder group may have a good reason to request blocking changes. A rotating board is designed to even out such influence over tech plans, as we would also like to avoid individuals having special privileges in the process.

Reply:
My first thought is that if you have blocking feedback, especially if significant, you probably _should_ have been added a as a stakeholder in the first place. So it doesn't bother me for us to suggest you add yourself as one? But curious what others think. 

Also intentionally aligning this with how we do PRs – if you have blocking feedback on a PR, do you usually add a review? Or just leave comments indicating the feedback is blocking?

Reply:
Similar to my [comment below](https://github.com/artsy/README/pull/483/files#r1030801473), do we need to clarify how we will resolve/escalate disagreements? While my hope is that most disagreements can be handled between the author(s) and the reviewer making blocking feedback, if we do allow reviewers to add themselves, I can envision a scenario where someone adds themselves as a reviewer and makes blocking feedback causing conflict. I think having a clear process for handling situations like this will help avoid tech plans which solicit strong opinions from becoming blocked.

Reply:
@mc-jones You bring up a good point. I'm hesitant to add more process in this iteration vs. dealing with (and learning from) conflicts as they arise. My hope is that by making the tech plans, feedback, and review statuses clearer, it'll be easier to discuss and understand different points of view, and will allow the tech plan writer to feel empowered to make calls (vs. delegating that responsibility elsewhere). I suspect most disagreements can be figured out with a face-to-face between those involved, and maybe we can rely on our standard approach to escalations (raising to your manager) if it becomes more sensitive. How does that sound?

Reply:
@sweir27 Sounds good!